platform/error/platform-error-impl/src/main/java/org/codice/ddf/platform/error/handler/impl/DefaultErrorHandler.java

```diff
@@ -65,29 +59,48 @@ public void handleError(
       String uri,
       HttpServletRequest request,
       HttpServletResponse response) {
-    initIndexHtml();
+    initErrorHtml();
 
-    String stack = ExceptionUtils.getFullStackTrace(throwable);
-
-    Map<String, String> jsonMap = new HashMap<>();
-    jsonMap.put("code", String.valueOf(code));
-    jsonMap.put("message", message);
-    jsonMap.put("type", type);
-    jsonMap.put("throwable", stack);
-    jsonMap.put("uri", uri);
-    String data = toJson(jsonMap);
-    String encodedBytes = Base64.getEncoder().encodeToString(data.getBytes(StandardCharsets.UTF_8));
-
-    String localIndexHtml = indexHtml.replace("WINDOW_DATA", "\"" + encodedBytes + "\"");
+    Map<String, String> attributesMap = new HashMap<>();
+    attributesMap.put("code", String.valueOf(code));
+    attributesMap.put("message", message);
+    attributesMap.put("uri", uri);
 
     response.setStatus(code);
     response.setContentType("text/html");
+
+    String output =
```

Comment:
📓 The old version of this used a combination of string replacement on the backend and handlebars on the frontend to generate the page. When moving to static generation, I had initially used the handlebars Java library that we use in other places in DDF, but the dependency tree grew larger than necessary for such simple string substitutions.